### ISSUE TITLE
Where is that skippy slider slider?

### ISSUE THREAD

**Issue by Eleksar387** [FartyPants/sd_script_skippy#1]
I am on latest AUTOMATIC. Installed it and restarted but I can't find any new slider. Where is it supposed to be?

**Comment by FartyPants** [comment]
It needs to go to script folder, then you need to select the Skippy the Step Kangaroo script in the list

**Comment by Eleksar387** [comment]
Unfortunately it does not appear there:
![image](https://user-images.githubusercontent.com/64690088/215283872-47512b01-bf28-4ce3-91d4-9cbe3a77f77a.png)

**Comment by FartyPants** [comment]
Did you put it to <sd-web-ui>/scripts/

**Comment by Eleksar387** [comment]
Ah! Just installed it using AUTOMATIC Extension > Install from URL. But I now copied it to the scripts folder. Thx!

**Comment by FartyPants** [comment]
Yeah, it isn't extension, the A1111 installer should tell you that it can't install anything. I may look into this how to make it an extension.
